### PR TITLE
[Fix] [Ready] Updates front page to fix animation and minor layout

### PR DESCRIPTION
**Pull request by caneruguz**
## Purpose
This PR updates the front page with the latest version from the designers. It fixes the issues with the network animation

## Changes
Replaced js in home-page.js with updated version
Replaced bits of css 

## Side effects
Most of the content is not changed so there shouldn't be any.

### CODE REVIEW

**Comment by sloria** [comment]
Once the travis build is passing for this, I will do 1st pass.

File: website/static/js/pages/home-page.js [hunk 1]
```diff
@@ -5,21 +5,29 @@ require('EasePack');
 require('vendor/youtube');
 
 // ANIMATION FOR FRONT PAGE
+
 $( document ).ready(function() {
-    $('#logo').removeClass('off');
-    $('.youtube').YouTubeModal({autoplay:1, width:640, height:480});
+  $('#logo').removeClass('off');
```
Comment (sloria) [inline website/static/js/pages/home-page.js]: > Use 4 spaces for indentation.

https://cosdev.readthedocs.org/en/latest/style_guides/javascript.html

File: website/static/js/pages/home-page.js [hunk 2]
```diff
@@ -5,21 +5,29 @@ require('EasePack');
...
-    var width;
-    var height;
-    var largeHeader;
-    var canvas;
-    var ctx;
-    var points;
-    var target;
-    var animateHeader = true;
+
+    var width, height, largeHeader, canvas, ctx, points, target, animateHeader = true;
```
Comment (sloria) [inline website/static/js/pages/home-page.js]: > Declare one variable per var statement.

https://github.com/felixge/node-style-guide#declare-one-variable-per-var-statement

File: website/static/js/pages/home-page.js [hunk 3]
```diff
@@ -165,22 +176,20 @@ $( document ).ready(function() {
...
     function Circle(pos,rad,color) {
-        var self = this;
+        var _this = this;
 
         // constructor
         (function() {
-            self.pos = pos || null;
-            self.radius = rad || null;
-            self.color = color || null;
+            _this.pos = pos || null;
```
Comment (sloria) [inline website/static/js/pages/home-page.js]: > Use self to save a reference to this.

https://cosdev.readthedocs.org/en/latest/style_guides/javascript.html

**Comment by sloria** [comment]
Please update this PR to be consistent with our style guide. 

Once that's done, I will do next pass.

**Comment by caneruguz** [comment]
Do you mean code style or UI style? 



—

**Comment by sloria** [comment]
code style

**Comment by caneruguz** [comment]
Ready for second pass @sloria , I hope this covers all style issues. I went through the file again after fixing the three above.

**Comment by haoyuchen1992** [comment]
@caneruguz  I found a new issue for the front-page change. On staging, the sign in button will disappear when showing the warning message.

Staging: 
screenshot 2015-07-31 17 17 40

Production:
screenshot 2015-07-31 17 17 48

**Comment by haoyuchen1992** [comment]
Another suggestion is that we may need to retain the big margin in production .  Occupying most of space on the screen is not good option. I read an article about this before.  

More discussion need be heard from you guys - @sloria @caneruguz 

Staging:
screenshot 2015-07-31 17 22 44

Production:
screenshot 2015-07-31 17 22 27

**Pull request merged**